src/nvim/lua/vim.lua

```diff
@@ -115,15 +115,17 @@ function vim._load_package(name)
   for _,path in ipairs(paths) do
     local found = vim.api.nvim_get_runtime_file(path, false)
     if #found > 0 then
-      return loadfile(found[1])
+      local f, err = loadfile(found[1])
+      return f or err
```

Comment:
`return f or error(err)` might work to override message.

Reply:
yeah, it looks better
<img width="1012" alt="Screen Shot 2020-12-08 at 02 11 55" src="https://user-images.githubusercontent.com/486807/101416406-df0bbd00-38fa-11eb-848b-74928a761cb0.png">